### PR TITLE
HIVEMALL-12: Add missing license headers

### PR DESCRIPTION
**Pull request by Lewuathe**
see: https://issues.apache.org/jira/browse/HIVEMALL-12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+<!-- 
+  Hivemall: Hive scalable Machine Learning Library
```
Comment (maropu) [inline README.md]: Need this title line?
I checked license headers in other apache projects and found all the same format they have;
ex.)
https://github.com/apache/incubator-systemml/blob/master/scripts/sparkDML.sh
https://github.com/apache/bahir/blob/master/bin/run-example
https://github.com/apache/flink/blob/master/tools/create_release_files.sh
...

Comment (Lewuathe) [inline README.md]: Should it be removed from existing headers as well? (cc: @myui )

Comment (myui) [inline README.md]: Yes, it should have a common license header.
https://github.com/myui/hivemall/blob/jira/HIVEMALL-12/resources/license-header.txt

**Comment by myui** [comment]
refer https://www.apache.org/legal/src-headers.html for source header rules.

**Comment by Lewuathe** [comment]
@myui Thanks. Did you already update all headers with [this commit](https://github.com/myui/hivemall/commit/6d2de2ccf5c84a7f7273e277900fdf6873df00fd)? If so we can close this PR, I think.

**Comment by myui** [comment]
@Lewuathe The branch just updated headers for `*.java` and `*.scala` .

**Comment by Lewuathe** [comment]
@myui Okay I'll recreate a PR for [jira/HIVEMALL-12](https://github.com/myui/hivemall/tree/jira/HIVEMALL-12) branch.

**Comment by myui** [comment]
@Lewuathe It might be able to use https://github.com/mycila/license-maven-plugin for *.sh and *.md.

I used it on jira/HIVEMALL-12.

**Comment by myui** [comment]
@Lewuathe updated *.md and *.sh license headers in 
https://github.com/myui/hivemall/tree/jira/HIVEMALL-12

So, I'll close this issue.

**Comment by Lewuathe** [comment]
👍

**Pull request closed without merging**